solutions/swb-reference/integration-tests/config/release-candidate-private.yaml

```diff
@@ -21,6 +21,9 @@ projectAdmin2PasswordParamStorePath: '/swb/release-candidate-private/pa2/passwor
 researcher1UserNameParamStorePath: '/swb/release-candidate-private/researcher1/email'
 researcher1PasswordParamStorePath: '/swb/release-candidate-private/researcher1/password'
 
+## Default external ID
+externalId: 'workbench'
```

Comment:
We don't want these to be published in the repo so these should be pointing to an ssm parameter name like we do for the above usernames and passwords.